module1/schedule.md

```diff
@@ -83,52 +78,56 @@ layout: page
 
 ## Week 4
 
-  * **Technical Lessons**
-    * [Event Manager - CSV Exploration](./projects/eventmanager)
-    * [Nested Iteration](./lessons/nested_iteration)
+  * **Lessons**
     * [Inheritance](./lessons/inheritance)
-    * [Mocks and Stubs](./lessons/mocks_stubs)
-
+    * [Nested Iteration](./lessons/nested_iteration)
+    * [Class Methods](./lessons/class_methods)
+    * Weekend Homework: [Event Manager - CSV Exploration](./projects/eventmanager)
 
-  * **Non-Technical Responsibilities**
+  * **Other Responsibilities**
     * Professional Development
-    * Gear Up #2
     * Accountabilibuddy Check-In
-    * Cohort Retro
+    * Week 4 with Jeff
+    * House Time
+    * Mix It Up Groups
 
   * **Deliverables**
-    * Independent Challenge #3
-    * Start [Group Project](./projects)
+    * Week 4 Independent Challenge
+    * Submit [Pair Project](./projects)
+    * Professional Development: LinkedIn Draft
+
+  * **Other**
+    * No Class Monday, February 20 | President's Day
 
 ## Week 5
 
-  * **Technical Lessons**
+  * **Lessons**
+    * [Event Manager - CSV Exploration](./projects/eventmanager)
     * [Design Principles](./lessons/design_principles)
+    * [Mocks and Stubs](./lessons/mocks_stubs)
     * [Ruby Object Model](./lessons/ruby_object_model)
-    <!-- * [Enumerables II](./lessons/enumerables_ii) -->
-
 
-  * **Non-Technical Responsibilities**
+  * **Other Responsibilities**
     * Professional Development
-    * Gear Up #3
     * Accountabilibuddy Check-In
-    * Cohort Retro
+    * House Time
+    * PD Discussion Groups
 
   * **Deliverables**
-    * Independent Challenge #4
+    * Week 5 Independent Challenge
     * Submit PD Survey
-    * Submit [Group Project](./projects)
-    * Start [Final Project](./projects)
+    * Start [Group Project](./projects)
 
 ## Week 6
 
-  * **Technical Lessons**
+  * **Lessons**
     * No new lessons
 
-  * **Non-Technical Responsibilities**
+  * **Other Responsibilities**
     * Professional Development
     * Final Cohort Retro
     * Accountabilibuddy Check-In
+    * Intro to B2
 
 
   * **Deliverables**
```

Comment:
Do we want to add the Final project due?

Reply:
Yes we definitely should! I'll add that.